apps/test/applabTest.js

```diff
@@ -111,11 +112,142 @@ describe('applab: getIdDropdown filtering modes', function () {
   it('does not accidentally pick up superset classes', function () {
     // Make sure searching for elements with class ".chart" does not also pick
     // up elements with class ".chart-friend"
-    assert.deepEqual(AppLab.getIdDropdownFromDom_(documentRoot, '.chart'), [
+    assert.deepEqual(Applab.getIdDropdownFromDom_(documentRoot, '.chart'), [
       { "display": '"chart9"', "text": '"chart9"' }
     ]);
-    assert.deepEqual(AppLab.getIdDropdownFromDom_(documentRoot, '.chart-friend'), [
+    assert.deepEqual(Applab.getIdDropdownFromDom_(documentRoot, '.chart-friend'), [
       { "display": '"image1"', "text": '"image1"' }
     ]);
   });
 });
+
+describe('hasDataStoreAPIs', function () {
```

Comment:
I would like to be able to run all applab mocha tests with a single command line. One option is to prefix all the descriptions in this file with "applab" so I can run `grunt mochaTest --grep applab`. What do you think? What do you do when you want to run all applab mocha tests?